### DISCUSSION TITLE
Why `vim.keymap.set({ "n" }, "x", "<Plug>(leap-forward-till)")` disables ability to use `<bs>` with multicursors.nvim?

### DISCUSSION THREAD

**Discussion by Andrew15-5** [ggandor/leap.nvim#199] [Q&A] [answered]
I've added `vim.keymap.set({ "n", "x", "o" }, "x", "<Plug>(leap-forward-till)")` and a few other keymaps for this plugin, but apparentely this blocks backspace from deleting one character to the left of the cursor when using [multicursors.nvim](https://github.com/smoka7/multicursors.nvim).

Then I found out that the culprit is `vim.keymap.set({ "n" }, "x", "<Plug>(leap-forward-till)")`. How `x` is linked to `<bs>`? Without this keymap (or without this plugin) the multicursors.nvim plugin works correctly.

**Comment by ggandor** [comment]
I have no idea unfortunately, but I guess this is has nothing to do with leap. Or if you map `x` to something else (custom), does multicursors work?

**Reply by Andrew15-5** [reply]
```lua
vim.keymap.set({ "n" }, "x", "<Plug>(leap-forward-to)")
vim.keymap.set({ "n" }, "x", "<cmd>echo 'hello'<cr>")
```
Either mapping is triggered when pressing `<bs>` in multicursor>insert mode. Since `x` has the ability to delete something, it's probably used somehow to simulate `<bs>`.

**Reply by Andrew15-5** [reply]
You can try using `docker` + `docker compose` + `just`:
1. Download [leap.zip](https://github.com/ggandor/leap.nvim/files/13466591/leap.zip)
2. `unzip leap.zip`
3. `cd leap`
4. `just`
5. press spacebar and then backspace
6. whitespace doesn't go away

If instead `just wo` is run, then the whitespace can be removed.

**Comment by ggandor** [comment, marked as answer]
This is a bug is in multicursors.nvim, just made a [PR](https://github.com/smoka7/multicursors.nvim/pull/75).

**Reply by Andrew15-5** [reply]
Thanks.
